### PR TITLE
Fixes for bugs found during manual use

### PR DESCRIPTION
**Pull request by nathanielmanistaatgoogle**
This is forward progress toward #3153.

### CODE REVIEW

**Comment by nathanielmanistaatgoogle** [comment]
Python is green.

**Comment by nicolasnoble** [comment]
Is this okay to merge ? Otherwise this will need to be re-done on the beta branch.

**Comment by nathanielmanistaatgoogle** [comment]
This is ready for merge.

File: src/python/grpcio/grpc/framework/core/_transmission.py [hunk 1]
```diff
@@ -246,7 +268,9 @@ def advance(self, initial_metadata, payload, completion, allowance):
 
   def timeout(self, timeout):
     """See _interfaces.TransmissionManager.timeout for specification."""
-    if self._transmitting:
+    if self._abort.kind is not _Abort.Kind.NOT_ABORTED:
```
Comment (soltanmm) [inline src/python/grpcio/grpc/framework/core/_transmission.py]: There's probably a nicer way to phrase this w/o double negatives, but nothing comes immediately to mind, so, meh.

**Pull request merged**